apps/docs/.eslintrc.json

```diff
@@ -9,23 +9,26 @@
         "@next/next/no-html-link-for-pages": "off"
     },
     "globals": {
```

Comment:
![image](https://github.com/gsoft-inc/wl-hopper/assets/361632/df3dd698-cd86-4a8a-8694-c0c261b68b97)
I don't think we should display the description here.